src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,22 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer {
```

Comment:
```suggestion
public class BulldozerProducer implements MachineProducer<Bulldozer> {
```
тобі потрібно тут параметризувати інтерфейс конкретним типом, інакше ти можеш проставити в методі get List<String> , List<Cat> і тд, а якщо при декларації класу параметризуєш його, то воно має сваритись, якщо обрати невалдіний тип для параметризації